snapcraft.yaml

```diff
@@ -45,11 +45,11 @@ description: |-
    - lxcfs.loadavg: Start tracking per-container load average [default=false]
    - lxcfs.cfs: Consider CPU shares for CPU usage [default=false]
    - lxcfs.debug: Increase logging to debug level [default=false]
+   - minio.path: Path to the minio binary to use with LXD [default=""]
```

Comment:
@masnax in a separate PR please can you update description of this setting to make it clear its the path to the directory containing the minio and mc commands thanks